### ISSUE TITLE
Bug: On custom LND images the server crashes when error message returns bytes

### ISSUE THREAD

**Issue by bjarnemagnussen** [jamaljsr/polar#465]
**Describe the bug**
When running a custom LND image and calling `SendCoins` with e.g. the following address `bc1pw508d6qejxtdg4y5r3zarvary0c5xw7kw508d6qejxtdg4y5r3zarvary0c5xw7k7grplx`, then the server crashes, as can be seen in the log (see at the end).

I think the reason is that the error message returned contains a byte value (`\u0001`), and _maybe_ for custom images the logger cannot handle interpreting byte values when printing?

**To Reproduce**
1. Create a custom LND image.
2. Call `SendCoins` gRPC with the following message:
```
{
  "addr": "bc1pw508d6qejxtdg4y5r3zarvary0c5xw7kw508d6qejxtdg4y5r3zarvary0c5xw7k7grplx"
}
```
3. The server crashes

**Expected behavior**
It should return an error "unsupported witness version: \u0001", which it does for the provided LND images.

**Screenshots**
Part of the log returned is:
```
2021-05-26 10:22:45.914 [INF] RPCS: [sendcoins] addr=bc1pw508d6qejxtdg4y5r3zarvary0c5xw7kw508d6qejxtdg4y5r3zarvary0c5xw7k7grplx, amt=0 BTC, sat/kw=12500, min_confs=1, send_all=false
runtime: goroutine stack exceeds 1000000000-byte limit
fatal error: stack overflow
runtime stack:
runtime.throw(0x158f938, 0xe)
	/usr/local/go/src/runtime/panic.go:774 +0x72
runtime.newstack()
	/usr/local/go/src/runtime/stack.go:1047 +0x6e9
runtime.morestack()
	/usr/local/go/src/runtime/asm_amd64.s:449 +0x8f
goroutine 885 [running]:
runtime.assertE2I2(0x136eda0, 0x1311de0, 0x22afd41, 0x0, 0x0, 0x0)
	/usr/local/go/src/runtime/iface.go:482 +0x9e fp=0xc021734358 sp=0xc021734350 pc=0x40c2de
fmt.(*pp).handleMethods(0xc0086336c0, 0x78, 0xc021734600)
	/usr/local/go/src/fmt/print.go:594 +0x88 fp=0xc021734568 sp=0xc021734358 pc=0x4df608
fmt.(*pp).printArg(0xc0086336c0, 0x1311de0, 0x22afd41, 0x78)
	/usr/local/go/src/fmt/print.go:713 +0x206 fp=0xc021734600 sp=0xc021734568 pc=0x4dfe46
fmt.(*pp).doPrintf(0xc0086336c0, 0x15ab599, 0x20, 0xc021734778, 0x1, 0x1)
	/usr/local/go/src/fmt/print.go:1030 +0x15b fp=0xc0217346e8 sp=0xc021734600 pc=0x4e380b
fmt.Sprintf(0x15ab599, 0x20, 0xc021734778, 0x1, 0x1, 0x136ed20, 0x4300ca)
	/usr/local/go/src/fmt/print.go:219 +0x66 fp=0xc021734740 sp=0xc0217346e8 pc=0x4dc6e6
github.com/btcsuite/btcutil.UnsupportedWitnessVerError.Error(...)
	/go/pkg/mod/github.com/btcsuite/btcutil@v1.0.3-0.20201208143702-a53e38424cce/address.go:26
github.com/btcsuite/btcutil.(*UnsupportedWitnessVerError).Error(0x22afd41, 0x1311de0, 0x22afd41)
	<autogenerated>:1 +0x93 fp=0xc021734798 sp=0xc021734740 pc=0x6721b3
```

**Desktop (please complete the following information):**
- OS: Linux
- Polar Version: v1.2.0
- Docker Version: v19.03.12
- Docker Compose Version: v1.29.1

**Additional context**
I have also made a custom v.0.12.0 LND image, which should be the same that is provided with Polar. Using the one from Polar does not raise this issue, however using my custom image does. Of course when using other addresses, which are valid or return string errors, both images work.

**Comment by jamaljsr** [comment]
Can you share the Dockerfile of your custom image?

**Comment by bjarnemagnussen** [comment]
Thanks for looking into it!

Actually, after taking a deep look (and realizing that by mistake when testing on the official LND repo I used the Dockerfile that pulls down the master commit and therefore always failed), I think this is a bug with LND.

I have identified commit lightningnetwork/lnd@a620ce36824af7aee6201d9056df38c2c87d7a3d, which updates btcutil dep, to introduce this bug.

I therefore close this issue and will open it on the corresponding LND repo.
